packages/headache/headache.1.06/opam

```diff
@@ -0,0 +1,45 @@
+opam-version: "2.0"
+
+license: "LGPL-2.0-only"
+
+synopsis: "Automatic generation of files headers"
+description: """
+Lightweight tool for managing headers in source code files. It can
+update in any source code files (OCaml, C, XML et al).
+"""
+
+authors: [
+  "Vincent Simonet"
+  # contributors
+  "Patrick Baudin"
+  "Mehdi Dogguy"
+  "François Pottier"
+  "Virgile Prevosto"
+  "Ralf Treinen"
+]
+
+maintainer: "Patrick Baudin"
+homepage: "https://github.com/Frama-C/headache/"
+bug-reports: "https://github.com/Frama-C/headache/issues"
+dev-repo: "git+https://github.com/Frama-C/headache.git"
+
+depends: [
+  "camomile"
+  "dune" {>= "1.6"}
+]
+
+build: [
+  [ "dune" "build" "-p" "headache" ]
+]
+
+install:  [
+  [make "INSTALLDIR=%{prefix}%/bin" "install"]
+  [make "DOC_INSTALLDIR=%{prefix}%/doc/headache" "install-doc"] {with-doc}
+]
```

Comment:
@vprevosto What is the reason for having an install section here? I'm not seeing anything in the source that isn't already installed by the regular `headache.install` file

Reply:
These lines was written during the build transition from `make` to `dune`.
It seems that they  were not removed when the `install`  section was added into the `dune` file.
Thanks.

Reply:
I have opened https://github.com/ocaml/opam-repository/pull/22675 to simplify it